src/views/CharacterListView.js

```diff
@@ -25,11 +28,20 @@ class CharacterListView extends React.Component {
   }
 }
 
-// our mapStateToProps needs to have two properties inherited from state
+// our mapStateToProps needs to have two properties
+//  inherited from state
 // the characters and the fetching boolean
+const mapStateToProps = state => {
+  return {
+    characters: state.charsReducer.characters,
+    fetching: state.charsReducer.fetching
```

Comment:
Nice job here! A lot of people struggled knowing that the characters are output into the store through the charsReducer! Take some time to think about what these two lines mean. What could it look like if you had two reducers?